### PR TITLE
COMMERCE-2383 Added "manualAdjusted" columns on order and order items

### PR DESCRIPTION
**Pull request by alessiorendina**

### CODE REVIEW

**Comment by alessiorendina** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test

File: commerce-service/service.xml [hunk 1]
```diff
@@ -284,6 +284,7 @@
 		<column name="orderStatus" type="int" />
 		<column name="printedNote" type="String" />
 		<column name="requestedDeliveryDate" type="Date" />
+		<column name="manualAdjusted" type="boolean" />
```
Comment (ethanbustad) [inline commerce-service/service.xml]: Hey Alessio, can we call this `manuallyAdjusted` instead? That sounds more correct to me.

**Review by ethanbustad — CHANGES_REQUESTED** [review]

File: commerce-service/service.xml (hunk 1 above)
Comment (alessiorendina) [reply commerce-service/service.xml]: Yes, I will change it. Thanks

**Review by ethanbustad — APPROVED** [review]

**Pull request merged**